src/solver/conv_hip_implicit_gemm_bwd_data_xdlops.cpp

```diff
@@ -64,15 +64,24 @@ struct CKArgs
 {
     CKArgs(const ProblemDescription& problem)
     {
-        N        = ProblemInterpreter::GetBatchN(problem);
-        K        = ProblemInterpreter::GetOutputChannelK(problem);
-        C        = ProblemInterpreter::GetInputChannelC(problem);
-        input    = {ProblemInterpreter::GetInputHeightHi(problem),
-                 ProblemInterpreter::GetInputWidthWi(problem)};
-        output   = {ProblemInterpreter::GetOutputHeightHo(problem),
-                  ProblemInterpreter::GetOutputWidthWo(problem)};
-        filter   = {ProblemInterpreter::GetFilterHeightY(problem),
-                  ProblemInterpreter::GetFilterWidthX(problem)};
+        G = ProblemInterpreter::GetGroupCountG(problem);
+        assert(G == 1);
```

Comment:
If this is the assumption, why do we need to divide `K` and `C` below?

Reply:
I copy pasted this code from grouped solver. Eventually I'd like to combine these CKArgs classes as much as I can.

Reply:
This assert shall not be required if everything is fine inside `isApplicable` function.

Or do we have to check everything everywhere?